apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/GetBucketOptions.cs

```diff
@@ -52,6 +52,17 @@ public sealed class GetBucketOptions
         /// </summary>
         public RetryOptions RetryOptions { get; set; }
 
+        /// <summary>
+        /// The bucket generation must be set (to retrieve a soft-deleted bucket) if soft_deleted is set to true.
+        /// </summary>
+        public long? Generation { get; set; }
+
+        /// <summary>
+        /// Set to true to retrieve a soft-deleted bucket. It will return the bucket metadata only if the bucket exists and is in a soft-deleted state.
+        /// The bucket generation must be set if soft_deleted is set to true.
```

Comment:
Let's use something closer to the API documentaiton:

```suggestion
        /// If true, the soft-deleted version of the bucket will be retrieved.
        /// If true, <see ref="Generation"  /> must be set.
```